### ISSUE TITLE
building docs broken since release of Sphinx 4

### ISSUE THREAD

**Issue by jugmac00** [jugmac00/flask-reuploaded#89]
```
❯ tox -e docs
GLOB sdist-make: /home/jugmac00/Projects/flask-reuploaded/setup.py
docs inst-nodeps: /home/jugmac00/Projects/flask-reuploaded/.tox/.tmp/package/1/Flask-Reuploaded-1.1.1.dev0.zip
docs installed: alabaster==0.7.12,attrs==21.2.0,Babel==2.9.1,certifi==2020.12.5,chardet==4.0.0,click==8.0.0,coverage==5.5,docutils==0.17.1,Flask==2.0.0,Flask-Reuploaded @ file:///home/jugmac00/Projects/flask-reuploaded/.tox/.tmp/package/1/Flask-Reuploaded-1.1.1.dev0.zip,idna==2.10,imagesize==1.2.0,iniconfig==1.1.1,itsdangerous==2.0.0,Jinja2==3.0.0,MarkupSafe==2.0.0,packaging==20.9,Pallets-Sphinx-Themes==1.2.3,pluggy==0.13.1,py==1.10.0,Pygments==2.9.0,pyparsing==2.4.7,pytest==6.2.4,pytest-cov==2.11.1,pytz==2021.1,requests==2.25.1,snowballstemmer==2.1.0,Sphinx==4.0.0,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==1.0.3,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.4,toml==0.10.2,urllib3==1.26.4,Werkzeug==2.0.0
docs run-test-pre: PYTHONHASHSEED='3812319761'
docs run-test: commands[0] | sphinx-build -b html docs _build/html
Running Sphinx v4.0.0
loading pickled environment... done

Exception occurred:
  File "/home/jugmac00/Projects/flask-reuploaded/.tox/docs/lib/python3.8/site-packages/pkg_resources/__init__.py", line 777, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (Jinja2 3.0.0 (/home/jugmac00/Projects/flask-reuploaded/.tox/docs/lib/python3.8/site-packages), Requirement.parse('Jinja2<3.0,>=2.3'), {'Sphinx'})
The full traceback has been saved in /tmp/sphinx-err-wfvq4oi0.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
ERROR: InvocationError for command /home/jugmac00/Projects/flask-reuploaded/.tox/docs/bin/sphinx-build -b html docs _build/html (exited with code 2)
_________________________________________________________________ summary __________________________________________________________________
ERROR:   docs: commands failed
```

**Comment by jugmac00** [comment]
Sphinx 4 does not play well with the Pallets theme.

**workaround**
There seems to be a fix on Sphinx master already.

**Comment by jugmac00** [comment]
fixed by #92
